### ISSUE TITLE
Epic: open design system V2 - The unification

### ISSUE THREAD

**Issue by nloureiro** [ethereum/ethereum-org-website#9431]
## Intro
This EPIC follows the [open design system V1](https://github.com/ethereum/ethereum-org-website/issues/6284)

<br>
<br>

## Summary

Design lives at the core of [[ethereum.org](http://ethereum.org/)](http://[ethereum.org](http://ethereum.org/)/). Our design principles, inspired by the ethos of the Ethereum ecosystem, guide our thinking and shape the content and features of our product. Similar to how our website content and code are open source, we believe the [ethereum.org](http://ethereum.org/) design should be open and collaborative.

When we started to work on the design system, we set to do all the work under one epic that would last for several quarters.

With the evolution of the file, we’ve decided to break it into 3 parts.

V1 - The foundations | V2 - The unification | V3 - The spacial pages
-- | -- | --
Base styles (colors + typography + ) Core foundations (icons + ) Basis components (avatars, tooltips, tags) Comum component like header footer Composed components (hero, modal) Base layout to all screens Markdown pages layout | Take the components used on different pages and unify the design under the guidelines. example: Search cards feedback widget accordeons charts and basic viz content grids proposals FAQ Product card Callouts Quiz | Take special pages, bring them into the Design system, and propose improvements if needed. Example: stablecoins find wallet events page language switcher stake pages hubs pages Tutorials page
<br>
<br>

#### Compare the difference in the approach. This is the main drive for each part

On the V1 we… | On the V2 we… | On the V3 we…
-- | -- | --
one big epic with various stories | one epic with quarter-scoped stories | one epic per page
developed by the core team | developed by core team + contributions | developed by core team + contributions
one Github issue with updates | one GitHub issue with a list of other issues to be worked on in parallel | one issue per epic
was working on the master file | worked on separate files and into the merged into master | worked on separate files and in the merged into master
Minimal Design changes made with quick design wins | incremental design changes by improving and unifying the components | strong and critical design approach on why we need custom-made solutions
Focus on base elements | Focus on re-usable elements | Focus on one of the solutions to specific problems
Goal to step up the base level | Goal to unify the experience | Goal to redesign
No storybook | includes the storybook integration (might be a blocker if the implementation affects the core master file ) | requires the storybook integration
acknowledge the existing design and experience | Improves the experience by unifying and redesigning some reusable components. | Rethinks the full design
no content is affected | very little content is affected besides the possible redesign of some flows and some hierarchies | might need content changes
driven by look and feel | driven by the look and feel, driven by behavior, and unify user experience | driven by user data

<br>
<br>

## Main goals

The main goal for this issue is to update the design system in several different priority tasks, and with this process, we will improve the Design System and fix/improve that the V1 is archived.

<br>
<br>

## Lists of Tasks

- [x] https://github.com/ethereum/ethereum-org-website/issues/9430
- [x] https://github.com/ethereum/ethereum-org-website/issues/9432
- [x] https://github.com/ethereum/ethereum-org-website/issues/9433
- [x] https://github.com/ethereum/ethereum-org-website/issues/9434
- [x] https://github.com/ethereum/ethereum-org-website/issues/9435
- [x] https://github.com/ethereum/ethereum-org-website/issues/9436
- [x] https://github.com/ethereum/ethereum-org-website/issues/9437
- [x] https://github.com/ethereum/ethereum-org-website/issues/9438
- [x] https://github.com/ethereum/ethereum-org-website/issues/9439
- [x] https://github.com/ethereum/ethereum-org-website/issues/9440
- [x] https://github.com/ethereum/ethereum-org-website/issues/9441

<br>
<br>

## Workflow

We prioritize a list of tasks, which might still need to be determined, but to start, we go with these tasks.
Each task has a Github issue associated with focusing the discussion on one platform that we can track and assign to the ones that want to work on the specific story.
Most of the design work is intended to be made on Figma, so please follow the link to each Figma file on the GitHub issue and read the guides on collaborating in Figma.

These are the main steps for each task.
1. #### Use a separate Figma file.
    - made by the team and liked here should be
2. #### Identify on which pages the component is used.
    - This can be made collaboratively and consists of screenshots/links to the pages where these components are already in use. Note that we can propose the unification of similar components in one if we find that the UX/UI justifies that’s the core unification goal of the Design system V2.
3. #### Identify the chakra component to be used as a base (if any)
    - We use chakra UI as a base framework, and if we can match the new design with an existing UI component in chakra, it’s a good call. If not, we can build one, but this should be something to identify now that will ease the implementation.
4. #### Work on the design.
    - This is where we build the new proposal that will be a new component in the design system, never forgetting the different screen sizes, behavior states, and themes.
5. #### Test and validate the new component on Figma.
    - Before closing the design, we need to test some real use cases. I suggested having some production screenshot as the base and building it on a Figma page with real content to test the live application of the new design.
6. #### Validation and feedback.
    - At this point, we are ready to validate and collect feedback. We should act upon this and have the best proposal possible.
7. #### Spec to handover to code.
    - This will be the final step, where we review the file from the perspective of a handover to the design system master file and to the code or storybook for code development.
8. #### Merge with the master Figma file of the Design system.
    - This step will be made by someone from the core team because there are some things that could be improved in how we can onboard external contributors to the Figma team. It’s a Figma thing that we can’t change for now.
9. #### Close the issue and publish the Design system update on the Figma community**.
    - For the last action, we need to close the issue and update the Design system on the Figam community file. At this point, another issue might need to be created to update the storybook and the code to reflect this new design.

<br>
<br>

**Comment by alex1092** [comment]
I would be keen to code these designs for you when they are ready to dev... is there a related epic for ds2 components that are ready to dev?

**Comment by nloureiro** [comment]
> I would be keen to code these designs for you when they are ready to dev... is there a related epic for ds2 components that are ready to dev?

Thank you for reaching out. Soon we will have an issue with that. @pettinarip will be leading the transition to code.
Stay tuned :)

**Comment by ocnoma** [comment]
hi @nloureiro, i'd like to take on the FAQ component task! what's the next step for claiming it?

**Comment by nloureiro** [comment]
> hi @nloureiro, i'd like to take on the FAQ component task! what's the next step for claiming it?

hey, thank you!
Can you please make a comment as the issue https://github.com/ethereum/ethereum-org-website/issues/9434
Then I will assign it to you.

🙏

**Comment by wackerow** [comment]
@nloureiro Curious the game plan for these:
- #9430
- #9436
- #9440
- #9441

Have the Figma changes created a blocker here?
